ui/components/src/stories/About.stories.mdx

```diff
@@ -0,0 +1,7 @@
+import { Meta } from '@storybook/addon-docs';
+
```

Comment:
These top-level "about" docs stories should be used to explain what each of the packages are for and how to use them. Not trying to fill these docs out as part of this PR, but putting them in place to show how to use them.